server/modules/auth/repository/authGroupRepository.js

```diff
@@ -61,12 +61,16 @@ const fetchSurveyGroups = async (surveyId, client = db) =>
 
 const fetchUserGroups = async (userUuid, client = db) =>
   await client.map(`
-    SELECT auth_group.*
-    FROM auth_group_user, auth_group
+    SELECT g.*, s.id as survey_id
+    FROM auth_group_user gu
+      JOIN auth_group g
+        ON g.uuid = gu.group_uuid
+      LEFT OUTER JOIN survey s
```

Comment:
you don't need left outer join, but did you really need to update this query?

Reply:
left outer join is not needed, you are right. I've changed the query to get the survey_id (used to log the userInviteAccept activity)

Reply:
no wait, the left outer join it's needed to fetch user groups associated to the systemAdmin user

Reply:
correct, you're right. but still, do you need to update this query ?

Reply:
Should I create another one for this?

Reply:
do we need another fetch function for this? how can I get the surveyId required to log the activities? with a SurveyRepository.fetchSurveyByUuid ?

Reply:
let's just remove the accept invitation activity log